styles/components/_masthead.scss

```diff
@@ -317,21 +317,62 @@
 // Selector Dropdown
 // ==================================================================
 .sprk-c-Masthead__selector-dropdown {
+  background-color: $sprk-white;
```

Comment:
Hey @RVMendoza, update this one to be a var please. Thank you!

Reply:
done!